qp_woltka/woltka.py

```diff
@@ -282,29 +154,92 @@ def woltka_to_array(directory, output, database_bowtie2,
              # includes ignoring files that have "0.00% overall alignment rate"
              # in their log file, which is when nothing aligns and is fine to
              # skip
-             "PROCESS=1; COUNTER=0; for f in `awk '{print $NF}' "
-             f'{output}/*.array-details`; do let COUNTER=COUNTER+1; '
-             "if [ ! -f ${f}*/free.biom ]; then if ! grep -xq "
-             "'0.00% overall alignment rate' *_${COUNTER}.err; "
-             "then PROCESS=0; fi; fi; done",
-             "if [ 1 -eq $PROCESS ]; then ",
+             # # "PROCESS=1; COUNTER=0; for f in `awk '{print $NF}' "
```

Comment:
Should these comments be deleted?

Reply:
No, we don't need them but those are the lines that are actually checking for failed samples so I would like to keep them until we have something in place to replace them.